src/QuestionDrawer.js

```diff
@@ -0,0 +1,59 @@
+import Icon from 'react-ionicons'
+import React from 'react'
+
+class QuestionDrawer extends React.Component {
+    render () {
+        const {
+            age,
+            diagnosed,
+            grade,
+            sex,
+            stage,
+            switchQuestion,
+        } = this.props
+        return (
+            <div className="variables flex-row">
+                <div onClick={() => switchQuestion('sex')} className="variable-switch-unanswered">
+                    <div className="input-unanswered">
```

Comment:
It looks like `input-unanswered` is special, is it even supposed to be here? All the questions seem to be in `unanswered-row` divs; this one is `input-unanswered` — maybe you don't need this div